main.py

```diff
@@ -1 +1,25 @@
 #Mad Libs Generator Project
+#Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+#All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
+#Displays the story based on the users input
+    print ("------------------------------------------")
```

Comment:
You could think about making one print statement and possibly a function

Reply:
> You could think about making one print statement and possibly a function

`a = """
text
text
text
"""

print(a)`